app/views/flood_risk_engine/enrollments/steps/show.html.erb

```diff
@@ -35,7 +35,15 @@ see the DigitalServicesCore::BackLink class -->
       <%= render step.to_s, form: form, step: step, f: f %>
 
       <div class="form-group">
-        <%= f.submit submit_button_text(form.enrollment), class: "button"  %>
+        <%=
+          f.submit(
```

Comment:
personally wouldn't change long established code, for no obvious benefit especially with the lack of decent feature and view specs but :ship: